dbt/models/qc/schema.yml

```diff
@@ -2,33 +2,6 @@ models:
   - name: qc.vw_change_in_ahsap_values
     description: '{{ doc("view_vw_change_in_ahsap_values") }}'
 
-  - name: qc.vw_incorrect_val_method
-    description: '{{ doc("view_vw_incorrect_val_method") }}'
-    config:
-      tags:
-        - test_qc_inquire
-    columns:
-      - name: revcode
-        data_tests:
-          # Equivalent to Res parcels not set to Cost Approach from FP Checklist
-          - accepted_values:
-              name: qc_vw_incorrect_val_method_revcode_eq_1
-              values: ['1']
-              additional_select_columns:
-                - taxyr
-                - parid
-                - class
-                - township_code
-                - who
-                - wen
-              config:
-                where: |
-                  CAST(taxyr AS int) BETWEEN {{ var('test_qc_year_start') }} AND {{ var('test_qc_year_end') }}
-                  AND class LIKE '2%'
-              meta:
-                table_name: aprval
-                description: revcode should be '1' for 200-class parcels
```

Comment:
This test got moved to the new `qc.vw_report_town_close_res_parcels_not_set_to_cost_approach` model.